packages/@atjson/document/src/attributes.ts

```diff
@@ -100,3 +100,15 @@ export function clone(attribute: any): NonNullable<any> {
     return attribute;
   }
 }
+
+export function removeUndefinedValuesFromObject(obj: any): any {
+  const newObj = { ...obj };
+  Object.keys(newObj).forEach(key => {
```

Comment:
Per our code style for this repository, we don't use `forEach`. https://atjson.condenast.io/docs/code-style#prefer-for-loops

This is for performance and diagnostics.

Reply:
oh ok cool. Updated this thanks 👍